pkg/hello/syncing.go

```diff
@@ -173,7 +182,7 @@ func NewHTTPSyncingClient(
 	if !ok {
 		return nil, errors.New("sync_server_address not found in pairing response metadata")
 	}
-	transport := NewHTTPClientSyncingTransport(syncServerAddress)
+	transport := NewHTTPClientSyncingTransport(syncServerAddress, 3*time.Second)
```

Comment:
Ideally this should be configurable through helm or extracted as const, I think it's ok for now.

Reply:
Agreed, leaving for now